examples/basic/verifying_nd/verifying_nd_1.py

```diff
@@ -0,0 +1,82 @@
+import desbordante
+import csv
+
+RED = '\033[31m'
+GREEN = '\033[32m'
+CYAN = '\033[1m\033[36m'
+ENDC = '\033[0m'
+
+def row_to_padded_string(row, widths):
+    return ''.join(field.ljust(width) for width, field in zip(widths, row))
+
+def print_table(filename: str, row_numbers = []):
+    with open(filename, newline='') as table:
+        rows = list(csv.reader(table, delimiter=','))
+
+    needed_rows = [rows[0]]
+    if len(row_numbers) > 0:
+        for row_num in row_numbers:
+            needed_rows.append(rows[row_num + 1])
+    else:
+        needed_rows = rows
+
+    column_widths = []
+    for col_num in range(len(needed_rows[0])):
+        max_len = max(len(row[col_num]) for row in needed_rows)
+        column_widths.append(max_len + 3)
+
+    header, *data_rows = needed_rows
+    print(row_to_padded_string(header, column_widths))
+    print('-' * sum(column_widths))
+    print('\n'.join(row_to_padded_string(row, column_widths) for row in data_rows))
+
+EXPIRED_PASSPORT_TABLE = 'examples/datasets/nd_verification_datasets/expired_passport.csv'
+VALID_PASSPORTS_TABLE = 'examples/datasets/nd_verification_datasets/valid_passports_1.csv'
+
+print(f'''{CYAN}This example illustrates the usage of Numerical Dependencies (NDs).
+Intuitively, given two sets of attributes X and Y, there is an ND from X to Y (denoted X -> Y, weight = k)
+if each value of X can never be associated to more than k distinct values of Y.
+For more information consult "Efficient derivation of numerical dependencies" by Paolo Ciaccia et al.
+{ENDC}''')
+
+print('Citizens of Arstozka can have no more than two documents: one Arstozka passport and one exit permit.')
+print('The following table contains records of some citizens\' documents:')
+
+print(CYAN, end='')
+print_table(EXPIRED_PASSPORT_TABLE)
+print(ENDC)
+
+algo = desbordante.nd_verification.algorithms.NDVerifier()
+algo.load_data(table=(EXPIRED_PASSPORT_TABLE, ',', True))
+algo.execute(lhs_indices=[0], rhs_indices=[1], weight=2)
+
+highlights = algo.highlights
+
+print('We need to validate these data')
+print('Let\'s run ND verification algorithm to check that every citizen has no more than two records:')
+print('\tND: [Name] -> [ID] with weight 2')
```

Comment:
lhs and rhs are sets, so I think that it is better to print '{Name} -> {ID}' instead of '[Name] -> [ID]' because it can be understood as a list.